### PR TITLE
RSPEC Update: Add code data-lang

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Related to #5346

### CODE REVIEW

File: analyzers/rspec/cs/S108_c#.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <p>Most of the time a block of code is empty when a piece of code is really missing. So such empty block must be either filled or removed.</p>
 <h2>Noncompliant Code Example</h2>
 <pre>
-for (int i = 0; i &lt; 42; i++){}  // Empty on purpose or missing piece of code ?
+<code data-lang="text">for (int i = 0; i &lt; 42; i++){}  // Empty on purpose or missing piece of code ?</code>
```
Comment (costin-zaharia-sonarsource) [inline analyzers/rspec/cs/S108_c#.html:4]: I think this should be `<code data-lang="csharp">`

Comment (pavel-mikula-sonarsource) [reply analyzers/rspec/cs/S108_c#.html:4]: We can't fix these, the root cause is in the RSPEC repo:
https://github.com/SonarSource/rspec/blame/master/rules/S108/noncompliant.adoc

Comment (pavel-mikula-sonarsource) [reply analyzers/rspec/cs/S108_c#.html:4]: Because the example is shared between multiple languages and the highlighting doesn't work as expected.

**Comment by pavel-mikula-sonarsource** [comment]
This change is not needed with rule-api 2.3

**Pull request closed without merging**